src/sql/run.py

```diff
@@ -400,8 +400,8 @@ def run(conn, sql, config, user_namespace):
                         conn.session.execution_options(isolation_level="AUTOCOMMIT")
                     except Exception as e:
                         print(
-                            "The database driver do not support \
-                            such execution option", e
+                            "The database driver do not support "
```

Comment:
don't support -> doesn't support
(same in test)

Reply:
> @edublancas what did we decide on this "autocommit" execution?

see the latest paragraph [here](https://github.com/ploomber/jupysql/issues/90#issuecomment-1420628820)

tl;dr; use the native autocommit if possible, otherwise fallback to the current implementation (issuing  a COMMIT command manually)

also @idomic FYI, this is what I proposed as a solution